front_end/panels/ai_chat/LLM/OpenRouterProvider.ts

```
@@ -144,7 +164,7 @@ export class OpenRouterProvider extends LLMBaseProvider {
         method: 'POST',
         headers: {
           'Content-Type': 'application/json',
-          'Authorization': `Bearer ${this.apiKey}`,
+          'Authorization': `Bearer ${this.getApiKey()}`,
```

ESLint

[error] Unnecessarily quoted property 'Authorization' found.

(@stylistic/quote-props)